examples/kubernetes/Makefile

```diff
@@ -0,0 +1,41 @@
+include ../../Makefile.defs
+
+ifeq ($(CILIUM_VERSION),)
+    CILIUM_VERSION = "$(shell cat ../../VERSION)"
+endif
+
+all: transform cilium.yaml
+
+%.sed:
+	for k8s_version in 1.7 1.8 1.9 1.10 1.11; do \
+	    mkdir -p $$k8s_version && \
+	    cd $$k8s_version && \
+	    sed -f transforms2sed.sed ../templates/v1/$@ | \
+	    sed s+__CILIUM_VERSION__+$(CILIUM_VERSION)+g > "$*"; \
+	    cd ..; \
```

Comment:
I'm not sure that it's any more clear, but if you run a subshell inside the loop (with `(blah)`) you won't need the `cd ..` and `cd -`.